felix/dataplane/linux/bpf_ep_mgr_test.go

```diff
@@ -317,7 +317,7 @@ var _ = Describe("BPF Endpoint Manager", func() {
 					"BPFConnectTimeLoadBalancingWorkaround": "enabled",
 				},
 				BPFPolicyDebugEnabled:  true,
-				BPFDisableGROForIfaces: []string{"wlp0s20f3", "virbr0", "docker0", "br-7ec0145f6b33"},
+				BPFDisableGROForIfaces: []string{"fake-iface1", "fake-iface2"},
```

Comment:
you will need to mock the ethtool if you want to test this. Another option is to do it in an fv test, where you can use the real ethtool to check that the value has changed.

Reply:
Thanks @tomastigera - replace unit test with FV test